### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

File: R/rope.R [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #'
 #' @details Statistically, the probability of a posterior distribution of being
 #'   different from 0 does not make much sense (the probability of it being
-#'   different from a single point being infinite). Therefore, the idea
+#'   different from a single point being 100%). Therefore, the idea
```
Comment (DominiqueMakowski) [inline R/rope.R]: I think infinite is correct here

File: vignettes/region_of_practical_equivalence.Rmd [hunk 2]
```diff
@@ -72,7 +72,7 @@ If the **full ROPE** is used (*i.e.*, 100\% of the HDI), then the null hypothesi
 
 # Sensitivity to the parameter's scale
 
-However, keep in mind that unlike indices of effect *existence* (such as the [`pd`](https://easystats.github.io/bayestestR/articles/probability_of_direction.html)), indices of effect *size*, such as the **ROPE percentage**, depend on the unit of its parameter, and can thus be easily changed *via* the scaling of the predictors.
+However, keep in mind that unlike other indices of effect **existence** (such as the [`pd`](https://easystats.github.io/bayestestR/articles/probability_of_direction.html)), **ROPE percentage** depend on the unit of its parameter, thus correct interpretation of the ROPE as representing a region of practical equivalence to zero is dependent on the scaling of the predictors.
```
Comment (DominiqueMakowski) [inline vignettes/region_of_practical_equivalence.Rmd]: In this case, maybe `unlike indices of effect *existence* the pd, indices of effect *significance*, such as the ROPE` so it the distinction is clearer

File: vignettes/region_of_practical_equivalence.Rmd [hunk 3]
```diff
@@ -120,7 +120,7 @@ p_direction(model)
 rope(model, ci=1)
 ```
 
-As you can see, by simply dividing the predictory by 100, we **drastically** changed (*we divided it by 100*) the conclusion related to the **percentage in ROPE** (which became very close to `0`): the effect could now be **interpreted as being significant**. Thus, we recommend, when reporting or reading ROPE results, to pay close attention to the unit of the predictors.
+As you can see, by simply dividing the predictor by 100, we **drastically** changed the conclusion related to the **percentage in ROPE** (which became very close to `0`): the effect could now be **interpreted as being significant**. Thus, we recommend paying close attention to the unit of the predictors when selecting the ROPE range (e.g., what coefficient would correspond to a small effect?), and when reporting or reading ROPE results.
```
Comment (DominiqueMakowski) [inline vignettes/region_of_practical_equivalence.Rmd]: in this case, it is not particularly the ROPE range (which is fixed in both cases and depends on the unit of the outcome) but the unit of the predictor

File: R/rope.R (hunk 1 above)
Comment (mattansb) [reply R/rope.R]: The probability of a single point is 0, so the complementary probability to that is 100-0=100%. No?

File: vignettes/region_of_practical_equivalence.Rmd (hunk 2 above)
Comment (mattansb) [reply vignettes/region_of_practical_equivalence.Rmd]: Oh yes! I like that! Maybe even something about clinical significance?

File: vignettes/region_of_practical_equivalence.Rmd (hunk 3 above)
Comment (mattansb) [reply vignettes/region_of_practical_equivalence.Rmd]: Yes, that's kind of the point I think I'm trying to make - the ROPE range should correspond to the predictors' units. How would you change this?

Comment (strengejacke) [reply vignettes/region_of_practical_equivalence.Rmd]: Not sure, but maybe my quote here helps: https://github.com/easystats/bayestestR/issues/144#issuecomment-495274448

File: R/rope.R (hunk 1 above)
Comment (strengejacke) [reply R/rope.R]: No, I don't think so, I think infinite is correct, or maybe undefined, but I think infinite probability is common in maths.

**Comment by mattansb** [comment]
I've heard of negative probability, but couldn't find any infinite
probability on Google 🤔

**Comment by DominiqueMakowski** [comment]
> I've heard of negative probability, but couldn't find any infinite probability on Google 🤔

Let's just say "the probability of a distribution of continuous values against a single point makes no sense" 😅

**Comment by mattansb** [comment]
Or maybe we can simply say: probability of a single point in a continuous
distribution of values is 0 / undefined?

**Comment by DominiqueMakowski** [comment]
"the probability of a single null value in a continuous distribution is 0"

**Comment by strengejacke** [comment]
Is it 0 or 1/Inf?

**Comment by strengejacke** [comment]
I remember something with Limes from math classes in school...

**Comment by DominiqueMakowski** [comment]
The probability of a single point is 0, therefore the probability of the other values (of the posterior distribution) being different from 0 (against a single null value of probability 0) is 1/0 = 💥 

I might be mistaken tho 😅

**Comment by mattansb** [comment]
Probability in a continuous distribution is the area under the curve between two points - the area between one point, is 0 (equal to the integral - so the change at a single point is 1/Inf, so either is correct, but I think the 0 is easier to understand?)

I think for the probability of the other values you need to subtract, not divide: 100-0 (or 100-1/Inf)=100%

**Comment by DominiqueMakowski** [comment]
I think "the probability of a null hypothesis single value in a continuous distribution is 0" is relatively clear

**Pull request merged**